tests/e2e/test.ts

```diff
@@ -0,0 +1,17 @@
+import { expect, test } from "@playwright/test"
+
+test("index page has expected title", async ({ page }) => {
```

Comment:
cool! I've never done this kind of testing before. love it!

Reply:
first time for me too. playwright is lovely - found it super easy to do. we'll need to write more tests here later after things are a little less in flux! could be wasted effort otherwise. by later I mean soon though! testing rocks!